### PR TITLE
feat: structuring event and command handlers separation

### PR DESCRIPTION
**Pull request by ruirochawork**
- Added Structuring event and command handlers separation
- Increased unit test coverage

image

### CODE REVIEW

**Review by pedrosousabarreto — APPROVED** [review]
Looks good to me!

File: packages/quoting-api-svc/src/service.ts [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+const BC_NAME = "quoting-bc";
+const APP_NAME = "quoting-api-svc";
+const APP_VERSION = packageJSON.version;
+
+const PRODUCTION_MODE = process.env["PRODUCTION_MODE"] || false;
+const LOG_LEVEL: LogLevel = (process.env["LOG_LEVEL"] as LogLevel) || LogLevel.DEBUG;
+
+const KAFKA_URL = process.env["KAFKA_URL"] || "localhost:9092";
+const MONGO_URL = process.env["MONGO_URL"] || "mongodb://root:mongoDbPas42@localhost:27017/";
```
Comment (elnyry-sam-k) [inline packages/quoting-api-svc/src/service.ts:72]: Comment from sonarcloud scan: "Make sure this MongoDB database password gets changed and removed from the code."

Please address this security issue and lets make the best use of the analysis from the available tooling. Analysis on this PR here: https://sonarcloud.io/summary/new_code?id=mojaloop_quoting-bc&pullRequest=17

File: packages/command-handler-svc/src/service.ts [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+const packageJSON = require("../package.json");
+
+const BC_NAME = "quoting-bc";
+const APP_NAME = "command-handler-svc";
+const APP_VERSION = packageJSON.version;
+const PRODUCTION_MODE = process.env["PRODUCTION_MODE"] || false;
+const LOG_LEVEL: LogLevel = process.env["LOG_LEVEL"] as LogLevel || LogLevel.DEBUG;
+
+const KAFKA_URL = process.env["KAFKA_URL"] || "localhost:9092";
+const MONGO_URL = process.env["MONGO_URL"] || "mongodb://root:mongoDbPas42@localhost:27017/";
```
Comment (elnyry-sam-k) [inline packages/command-handler-svc/src/service.ts:93]: Issue reported by sonarcloud with comment: "Make sure this MongoDB database password gets changed and removed from the code."

Full analysis on this PR is here: https://sonarcloud.io/summary/new_code?id=mojaloop_quoting-bc&pullRequest=17

**Comment by elnyry-sam-k** [comment]
hi @ruirochawork, thanks for the PR.

Couple of comments:
1. This PR has 84 files changed, please ensure PRs are manageable for better reviews and quality reasons (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#write-small-prs)
2. Which issue on Mojaloop project board is this against? Whats the acceptance criteria for this?

**Pull request merged**